modules/apps/journal/journal-web/src/main/resources/META-INF/resources/article/schedule.jsp

```diff
@@ -31,7 +43,40 @@ JournalEditArticleDisplayContext journalEditArticleDisplayContext = new JournalE
 <liferay-ui:error exception="<%= ArticleExpirationDateException.class %>" message="please-enter-a-valid-expiration-date" />
 
 <div class="schedule">
-	<aui:input formName="fm1" name="displayDate" wrapperCssClass="mb-3" />
+	<div class="form-group display-date mb-3 input-Date-wrapper">
+		<label class="control-label"><liferay-ui:message key="display-date" /></label>
+
+		<div class="form-group-autofit">
+			<div class="form-group-item">
+				<liferay-ui:input-date
+					dayParam="displayDateDay"
+					dayValue="<%= cal.get(Calendar.DATE) %>"
+					disabled="<%= false %>"
+					formName="fm1"
+					monthParam="displayDateMonth"
+					monthValue="<%= cal.get(Calendar.MONTH) %>"
+					name="displayDate"
+					nullable="<%= editingDefaultValue %>"
```

Comment:
@locpham97. The variable `editingDefaultValue` is unnecessary. Should use `journalEditArticleDisplayContext.getClassNameId()` directly.